.travis.yml

```diff
@@ -13,6 +13,16 @@ matrix:
   allow_failures:
     - rust: nightly
   fast_finish: true
+install:
+  - |
+    if [[ "$TRAVIS_RUST_VERSION" = "1.25.0" ]]; then
+      rustup target add aarch64-apple-ios armv7-apple-ios armv7s-apple-ios x86_64-apple-ios i386-apple-ios &&
```

Comment:
That's a lot of targets.  Do we actually run tests on all of these targets?

Reply:
We build a universal library that contains all of the targets so we do need to add all of them.